### PR TITLE
Adding option to set global toggles.

### PR DESCRIPTION
**Pull request by ghost**
This small change adds option to set some simple toggles in the global portion of haproxy configuration. This is needed for some smaller, one-of-a-kind fixes, like this one: https://github.com/FireBurn/haproxy/commit/f1e82e94edd96ce325e79ea8e92b6eae125ab37d where creating and maintaining a list would be very burdensome.

This could be achieved by misusing global tunes (just create a tune with empty value), but that approach looks like a really ugly hack.

### CODE REVIEW

File: templates/etc/haproxy/haproxy-global.cfg.j2 [hunk 1]
```diff
@@ -78,4 +78,9 @@ global
 {% endif %}
 {% if haproxy_global_server_state_base is defined %}
     server-state-base {{ haproxy_global_server_state_base }}
+{% endif %}
+{% if haproxy_global_toggles is defined and haproxy_global_toggles|length %}
```
Comment (goldyfruit) [inline templates/etc/haproxy/haproxy-global.cfg.j2]: Please add the the spaces between the `|`, if not `ansible-lint` will complain :laughing:

```
{% if haproxy_global_toggles is defined and haproxy_global_toggles|length %}
```
To:
```
{% if haproxy_global_toggles is defined and haproxy_global_toggles | length %}
```

Comment (ghost) [reply templates/etc/haproxy/haproxy-global.cfg.j2]: Hi! do I need to change it like this in whole file (see lines 72 or 67 for example)? It's fine by me, just don't want to do it twice :)

Comment (goldyfruit) [reply templates/etc/haproxy/haproxy-global.cfg.j2]: > Hi! do I need to change it like this in whole file (see lines 72 or 67 for example)? It's fine by me, just don't want to do it twice :)

If you found some others yes please :+1:

**Comment by goldyfruit** [comment]
Thanks @akowalik-transporeon :+1:

File: templates/etc/haproxy/haproxy-global.cfg.j2 (hunk 1 above)
Comment (ghost) [reply templates/etc/haproxy/haproxy-global.cfg.j2]: Alright, i think i got all of them now.

**Pull request merged**